rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java

```diff
@@ -629,7 +629,7 @@ private BlockResult executeForMiningAfterRSKIP144(
             }
 
             Optional<Long> sublistGasAccumulated;
-            if (tx.isRemascTransaction(txindex, transactionsList.size())) {
+            if (tx.isRemascTransaction(txindex-1, transactionsList.size())) {
```

Comment:
Instead of substracting to txIndex, txIndex should be incremented at the end of the method.